### DISCUSSION TITLE
A few issues to build DockerFile from "main" branch on Windows

### DISCUSSION THREAD

**Discussion by Ming-D-BigData** [apache/airflow#24358] [General] [answered]
I checkout today's "main" branch(hash: 23fb6635e...), trying to build the image on Windows. But I ran into 2 issues.
1) The code can't be fully checked out on Windows due to file path issue.
![image](https://user-images.githubusercontent.com/58782082/172931879-071b740d-faff-478f-8c95-ba396394e070.png)
2) Syntax error in scripts when running "docker build"
![image](https://user-images.githubusercontent.com/58782082/172932150-0da01543-2420-4301-ae85-05bca2c8a047.png)

Is there any Windows (Docker Desktop) specific branch?

**Comment by potiuk** [comment, marked as answer]
No. Airflow is not buildable or checkoutale  on Windows filesystem. But we are slowly getting there. We are counting on people ike you who run Windows and fix any errors there. PRs are most welcome! An example of improving witndows compatibility with `breeze` development environment is https://github.com/apache/airflow/pull/24098 - and you could provide similar fixes.

Feel free! This is the fastest way to add Windows compatibility, otherwise you will have to wait until somone else will improve it - but there are no guarantees for that.
